spec/models/ip_spec.rb

```diff
@@ -1,51 +1,56 @@
 describe Ip do
+  subject(:ip_address) { described_class.new }
+
   it { is_expected.to belong_to(:location) }
+  it { is_expected.to validate_presence_of(:address) }
+  it { is_expected.to validate_uniqueness_of(:address) }
+
+  context "when validating" do
```

Comment:
"when validating address" might be better. We are only validating the address in this context.